### DISCUSSION TITLE
Rails 7 support?

### DISCUSSION THREAD

**Discussion by ylluminate** [solidusio/solidus#4233] [New Features or Ideas]
What are we looking at for the timeline of Rails 7 support?

**Comment by waiting-for-dev** [comment]
That's being addressed at https://github.com/solidusio/solidus/pull/4220

**Reply by ylluminate** [reply]
Thanks so much for pointing us there. 🙇🏻‍♂️

**Reply by yshmarov** [reply]
@waiting-for-dev 
Is there a release planned? 🙏 
it's been 3 months since the PR was released with 4 months since last solidus release 😞

**Reply by jarednorman** [reply]
You're mistaken about when the last release was, but yes, we do plan to release a new version soon.

**Reply by waiting-for-dev** [reply]
Hey, @yshmarov, we're wrapping everything up. Stay tuned; this version will bring amazing stuff beyond supporting Rails 7 :partying_face:
